### PR TITLE
[OSDOCS#9850]: UI doc updates for longer cluster name feature

### PR DESCRIPTION
**Pull request by AedinC**
Version(s):
4.15+

Issue:
[OSDOCS-9850](https://issues.redhat.com/browse/OSDOCS-9850)

Link to docs preview:

**OSD Docs:**
- [Creating a cluster on AWS with CCS](https://72846--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-an-aws-cluster#osd-create-aws-cluster-ccs_osd-creating-a-cluster-on-aws)
- [Creating a cluster on GCP with CCS](https://72846--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-a-gcp-cluster#osd-create-gcp-cluster-ccs_osd-creating-a-cluster-on-gcp)
- [Creating a cluster on GCP with Google Cloud Marketplace](https://72846--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-a-gcp-cluster#osd-create-cluster-gcp-account_osd-creating-a-cluster-on-gcp)
- [Creating a cluster on GCP with Red Hat Marketplace](https://72846--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-a-gcp-cluster#osd-create-cluster-rhm-gcp-account_osd-creating-a-cluster-on-gcp)

**ROSA Docs:**
- [Creating a cluster with customizations by using OpenShift Cluster Manager](https://72846--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations#rosa-sts-creating-cluster-customizations-ocm_rosa-sts-creating-a-cluster-with-customizations)
- [Creating a ROSA cluster with AWS STS using the default auto mode](https://72846--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_getting_started/rosa-quickstart-guide-ui#rosa-quickstart-creating-a-cluster)
- [What’s new with Red Hat OpenShift Service on AWS](https://72846--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_release_notes/rosa-release-notes)

Peer review:
- [x] Peer reviewer has approved this change.

SME review:
- [x] SME has approved this change.

QE review:
- [x] QE has approved this change in JIRA issue.  

Additional information:

### CODE REVIEW

File: rosa_release_notes/rosa-release-notes.adoc [hunk 1]
```diff
@@ -27,6 +27,9 @@ For more information on region availabilities, see xref:../rosa_architecture/ros
 * **ROSA CLI update.** The ROSA CLI (`rosa`) was updated to a new version. For information about what has changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.36[ROSA CLI release notes]. For more information about the ROSA CLI (`rosa`), see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc#rosa-about_rosa-getting-started-cli[About the ROSA CLI].
 
 * **Log linking is enabled by default.** Beginning with {product-title} 4.15, log linking is enabled by default. Log linking gives you access to the container logs for your pods.
+* **Longer cluster names enhancement.** You can now specify a cluster name that is longer than 15 characters. For cluster names that are longer than 15 characters, you have the option to customize the domain prefix for the cluster URL by using the `domain-prefix` flag in the ROSA CLI (`rosa`) or by selecting the **Create custom domain prefix** checkbox in the {hybrid-console}. For more information, see xref:../cli_reference/rosa_cli/rosa-manage-objects-cli.adoc#rosa-create-cluster-command_rosa-managing-objects-cli[Managing objects with the ROSA CLI].
```
Comment (kcarmichael08) [inline rosa_release_notes/rosa-release-notes.adoc:30]:
```suggestion
* **Longer cluster names enhancement.** You can now specify a cluster name that is longer than 15 characters. For cluster names that are longer than 15 characters, you can customize the domain prefix for the cluster URL by using the `domain-prefix` flag in the ROSA CLI (`rosa`) or by selecting the **Create custom domain prefix** checkbox in the {hybrid-console}. For more information, see xref:../cli_reference/rosa_cli/rosa-manage-objects-cli.adoc#rosa-create-cluster-command_rosa-managing-objects-cli[Managing objects with the ROSA CLI].
```

**Review by kcarmichael08 — COMMENTED** [review]
One tiny suggestion; lgtm!

File: rosa_release_notes/rosa-release-notes.adoc (hunk 1 above)
Comment (AedinC) [reply rosa_release_notes/rosa-release-notes.adoc:30]: Thanks @kcarmichael08!

**Comment by AedinC** [comment]
@KevinFCormier @AkashKanni can you please review at your earliest convenience? Thanks.

**Review by DcooperRH — APPROVED** [review]

**Pull request merged**